build-tools/packages/build-cli/src/commands/generate/assertTags.ts

```diff
@@ -336,6 +341,7 @@ function getAssertMessageParams(
 			const args = call.getArguments();
 			if (args.length >= messageIndex && args[messageIndex] !== undefined) {
 				const messageArg = args[messageIndex];
+				assert(messageArg !== undefined, "messageArg is undefined in getAssertMessageParams");
 				messageArgs.push(messageArg);
 			}
```

Comment:
Do it in two steps since it is already testing for undefined. Like:
```typescript
			if (args.length >= messageIndex) {
				const messageArg = args[messageIndex];
				if (messageArg !== undefined) {
					messageArgs.push(messageArg);
				}
			}
```
Of course, this looks wrong to start with - the index test appears off. I think it should be:
```typescript
			if (args.length > messageIndex) {
```